index.js

```diff
@@ -51,6 +52,9 @@ var Omegle = function () {
 		}
 		request(options, function (error, response, body) {
 			if (!error && response.statusCode == 200) {
+				if(!working_servers.indexOf(url)>0){
```

Comment:
Oh that I know, I was talking about the syntax. `indexOf` returns -1 if the given element is not found. In what you're doing, it will return true if the `url` is present on 0th index (which is not what we want). Let's say `working_servers = ['someServer']` and `url='someServer'`. Now, when you'll check that with your condition, it will return true and push the `url` in `working_servers` (even though that server already exists). Something like `if(working_servers.indexOf(url)<0)` should work.

And one more thing, could you please use camelCase for naming variables instead of snake_case? I'll merge your commit after that. :)